src/chef-server-ctl/plugins/upgrade.rb

```diff
@@ -22,19 +23,19 @@ def parse(args)
     @options.chef11_admin_client_key = "/etc/chef-server/admin.pem"
 
     opt_parser = OptionParser.new do |opts|
-      opts.banner = "Usage: chef-server-ctl upgrade [options]"
-      opts.banner = opts.banner << "\n Options only apply to open source Chef 11 server to Chef 12 server upgrades."
-      opts.banner = opts.banner << "\n If upgrading from Enterprise Chef 11 server to Chef 12 server no options are needed."
+      opts.banner = "Usage: #{ChefUtils::Dist::Server::SERVER_CTL} upgrade [options]"
+      opts.banner = opts.banner << "\n Options only apply to open source #{ChefUtils::Dist::Server::PRODUCT} 11 to #{ChefUtils::Dist::Server::PRODUCT} 12 upgrades."
+      opts.banner = opts.banner << "\n If upgrading from Enterprise #{ChefUtils::Dist::Server::PRODUCT} 11 to #{ChefUtils::Dist::Server::PRODUCT} 12 no options are needed."
 
       opts.on("-y", "--yes", "Skip confirmation") do |y|
         @options.skip_confirmation = y
       end
 
-      opts.on("-o", "--org-name [name]", String, "The name of the Chef 12 organization to be created. It must begin with a lower case letter or digit; can only have lower case letters, digits, hyphens, and underscores and must be between 1 and 255 characters long (Will ask interactively if not passed).") do |n|
+      opts.on("-o", "--org-name [name]", String, "The name of the #{ChefUtils::Dist::Infra::Org} 12 organization to be created. It must begin with a lower case letter or digit; can only have lower case letters, digits, hyphens, and underscores and must be between 1 and 255 characters long (Will ask interactively if not passed).") do |n|
```

Comment:
Does not look like there is ChefUtils::Dist::Infra::Org, did you intend to use ChefUtils::Dist::Server::PRODUCT instead?

Reply:
Thanks! Yeah I don't know what I was trying to do here, but I just changed (and will shortly commit) this to: `ChefUtils::Dist::Infra::SHORT.capitalize`

Reply:
Unless you prefer `ChefUtils::Dist::Server::PRODUCT `

Reply:
Any changes to chef-utils will not reflect in server unless the version of chef client is updated in server. So perhaps for now we could use what is available if that seems appropriate?

Reply:
Do you care if the wording changes from `The name of the Chef 12 organization` to `The name of the Chef Infra Server 12 organization`?

Using `ChefUtils::Dist::Infra::SHORT.capitalize` feels a bit awkward, but it does leave the text the same...